R/parse_roxygen.R

```diff
@@ -161,25 +161,35 @@ parse_roxygen.r <- function(text){
 #' @importFrom utils capture.output
 #' @noRd
 parse_roxygen.r_test_code <- function(text){
-
   roxyblocks <- roxygen2::parse_text(text,env = NULL)
   roxyblocks <- cleanup_section_last_update(roxyblocks)
 
-  roxyblocks <- lapply(roxyblocks,function(block){
+  roxyblocks <- lapply(roxyblocks,function(block, file){
+    test <- tryCatch({
+      as.list(block$call)[[2]]
+    }, error = function(e){
+
+      roxygen2::block_get_tag(block, "title")$val
+    })
```

Comment:
I thought we'd discussed flipping that? look for the title. if not, fall back on test description? 

The way test_that was written, it requires both a desc and the code, so I don't think the there can be a failure to fall back to title.

Reply:
I see now in the test the purpose now for this ordering. Maybe we should add more explicit testing to make sure that deprecated tests have the call NULL?